fftools/ffmpeg.c

```diff
@@ -1743,19 +1743,33 @@ static void print_report(int is_last_report, int64_t timer_start, int64_t cur_ti
     float tr;
     int64_t reset_size;
     double cur_bitrate;
-    static int64_t transcode_total = 0;
-    static int64_t transcode_max = 0;
     static int64_t transcode_reset_time = 0;
-    static int transcode_reset_frame_number = 0;
-    static int64_t transcode_reset_total = 0;
-    static int64_t transcode_us_last = 0;
-    static float prev_transcode_pf = 0.0;
-    static int64_t prev_transcode_max = 0;
-
-    if(transcode_us > 0) {
-        transcode_total += transcode_us;
-        transcode_max = transcode_us > transcode_max ? transcode_us : transcode_max;
-        transcode_us_last = transcode_us;
```

Comment:
Can't most of these be uints?

Reply:
using same type as ffmpeg uses so dont need to be casting